ons-spark/spark-app/spark_application_and_ui.md

```diff
@@ -0,0 +1,429 @@
+# Spark Application and UI
+
+This article introduces the structure of a Spark application and how the structure relates to the various pages within the application monitoring interface, the Spark UI. 
+
+The Spark UI is used to monitor the status and resource consumption of your Spark cluster and is the main tool for troubleshooting slow Spark code. 
+
+Understanding the structure of a Spark application helps to understand how Spark works so you can begin to think differently as you code to get the most out of Spark.
+
+## Spark Application Overview
+A Spark application has an underlining structure and learning about this structure will help us
+1. understand the difference between a narrow transformation, wide transformation and an action, 
+2. navigate around the Spark UI
+
+Below is a diagram of the Spark application hierarchy.
```

Comment:
I'd add a colon: at the end of line 10 and 14 as you're introducing a list. You don't need the comma at the end of line 11 either